src/modules/user/user.usecase.ts

```diff
@@ -518,4 +555,36 @@ export class UserUseCases {
       password,
     });
   }
+
+  async updatePassword(
```

Comment:
The current endpoint for this on drive-server is [here](https://github.com/internxt/drive-server/blob/dca8b2b42d1510cd30f71efd6e67e0c504307856/src/app/routes/user.js#L11) you will see that `currentPassword`, `newPassword`, and `newSalt` are encrypted on transit, so you have to decrypt them before using them.